lxd/storage/utils.go

```diff
@@ -972,7 +972,7 @@ func volumeIsUsedByDevice(vol api.StorageVolume, instanceType instancetype.Type,
 // VolumeUsedByProfileDevices finds profiles using a volume and passes them to profileFunc for evaluation.
 // The profileFunc is provided with a profile config, project config and a list of device names that are using
 // the volume.
-func VolumeUsedByProfileDevices(s *state.State, poolName string, projectName string, vol *api.StorageVolume, profileFunc func(profileID int64, profile api.Profile, project api.Project, usedByDevices []string) error) error {
+func VolumeUsedByProfileDevices(s *state.State, poolName string, projectName string, vol *api.StorageVolume, profileFunc func(profile api.Profile, project api.Project, usedByDevices []string) error) error {
```

Comment:
I think this variable is shadowed by https://github.com/canonical/lxd/pull/15719/files#diff-33511e3a6e5abae8b54480aa23c933de0ec50584842b582286029955063c389dL1051